index.html

```diff
@@ -79,7 +80,7 @@ <h2><b>Tonic</b></h2>
           <li><a  class="button" href="#">CSS</a> </li>
           <li><a  class="button" href="#">javaScript</a></li>      
         </ul>
-        <button onclick="togglePopup1()" class="normalbutton seeproject">See Project</button>     
+        <button id="modalButton-1" class="normalbutton seeproject">See Project</button>     
```

Comment:
- [ ] Great job so far 👍 But I noticed that your project section is not loading the data from the array with project information stored 😢 . This projects section needs to be created dynamically with the information from the projects array you already created and use in your modal, note that all the HTML of the section is created when the page loads. Please remove the existing HTML code for the projects list and use JavaScript to load the information from the array 👍 .